nixos/modules/services/web-apps/tt-rss.nix

```diff
@@ -466,34 +468,32 @@ let
       '';
     } else {};
 
-    # TODO: Re-enable after https://github.com/NixOS/nixpkgs/pull/15862 is merged
-
-    # services.nginx.virtualHosts = if cfg.virtualHost == "${virtualHostName}" then {
-    #   "${virtualHostName}" = {
-    #     root = "${root}";
-    #     extraConfig = ''
-    #       access_log  /var/log/nginx-${virtualHostName}-access.log;
-    #       error_log   /var/log/nginx-${virtualHostName}-error.log;
-    #     '';
+    services.nginx.virtualHosts = mkIf (cfg.virtualHost != null) {
+      "${cfg.virtualHost}" = {
+        root = "${cfg.root}";
+        extraConfig = ''
+          access_log  /var/log/nginx-${cfg.virtualHost}-access.log;
+          error_log   /var/log/nginx-${cfg.virtualHost}-error.log;
```

Comment:
I'd personally not expect this, as all my other nginx logs go to systemd journal.